### ISSUE TITLE
Issue with stepping through history via up/down arrows

### ISSUE THREAD

**Issue by monoblaine** [dahlbyk/posh-git#271]
Suppose that you've executed the following commands respectively:

1. `git status`
2. `git difftool`
3. `git commit -a -m "foo bar"`
4. `git push`

So the last command you've executed is the 4th one. Now press the up arrow 4 times and get `git status` and press `Enter` to execute it. If you press the down arrow after that, you'll get `git difftool` (The next item following `git status`).

For some reason, this feature got broken after I've installed posh-git. Now, pressing the up arrow always starts from the most recent command in history. Even if you execute a command from the history, pressing the down arrow afterwards gives nothing. You still have to press the up arrow and start from the beginning.

Is there any way to fix this?

**Comment by dahlbyk** [comment]
Hrm...I'm not able to reproduce this. You can't see my keystrokes, but my last two commands are from hitting Down then Enter in succession.

![up-down-demo](https://cloud.githubusercontent.com/assets/133987/14024830/1a6b6050-f1b9-11e5-8d71-b35259836760.gif)

**Comment by monoblaine** [comment]
Sorry, it appears to be my fault. The fact is, the 32 bit version of the PowerShell was normal, but the 64 bit version had this problem. Also, 32 bit version's ExecutionPolicy was set to "Restricted" mode. So there was no posh-git in the 32 bit PowerShell, which made me think the problem was with the posh-git itself. Without trying uninstalling posh-git and checking whether the problem still existed, I've filed an issue here. After I'd seen your comment, I've uninstalled posh-git only to realize the problem was still there.

For the next two hours I frantically and hopelessly tried understanding what was wrong (randomly uninstalling modules, running PowerShell in admin mode, deleting random keys from regedit), I don't know why and how but on of those steps fixed it.
